nixos/modules/services/networking/consul.nix

```
@@ -99,6 +99,7 @@ in
 
       extraConfig = mkOption {
         default = { };
+        type = types.attrsOf types.str;
```

I tried running the consul NixOS test locally and it failed cuz not everything is a `types.str`.

```suggestion
        type = types.attrsOf types.anything;
```